tests/features/glb-models/glb-models02-local-model.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { Circuit } from "tscircuit"
+import { test, expect } from "bun:test"
+import { convertCircuitJsonToGltf } from "../../../lib"
+import { getBestCameraPosition } from "../../../lib/utils/camera-position"
+import { renderGLTFToPNGBufferFromGLBBuffer } from "poppygl"
+import dip4ModelUrl from "./assets/dip4.glb"
+
+test("glb-models02-local-model", async () => {
+  const circuit = new Circuit()
+  circuit.add(
+    <board width="25" height="25">
+      <chip
+        footprint="dip4"
+        name="U1"
+        cadModel={{
+          glbUrl: dip4ModelUrl,
```

Comment:
so you're using bun's loader here which is different than the loader in eval (a custom loader)

I think we need to configure bun's loader to do `file://` as the url which is fetch-compatible